ImageNet/train/dawn_resnet50_18minutes.json

```diff
@@ -0,0 +1,977 @@
+{
+    "version": "v1.0",
+    "author": "Yaroslav Bulatov, Andrew Shaw, Jeremy Howard",
+    "authorEmail": "yaroslavvb@gmail.com",
+    "framework": "PyTorch 0.4.1",
+    "codeURL": "https://github.com/diux-dev/imagenet18/tree/59a8f25171fb8cede51db9187a32fc8f802384a0",
+    "model": "ResNet-50",
+    "hardware": "16 p3.16xlarge (AWS)",
+    "costPerHour": 391.68,
+    "timestamp": "2018-09-06",
+    "misc": {
+        "model implementation": "https://github.com/diux-dev/imagenet18/blob/086675a6df3d468e89c651ae4c75f31e5b3f381d/training/resnet.py",
+        "backend": "NCCL 2.2.13+cuda9.2",
+        "optimizer": "SGD with Momentum",
+        "momentum": 0.9,
+        "weightDecay": 0.0001,
+        "schedule overview": "Base learning rate lr=1.88, schedule consists of several linear scaling segments as well as manual learning rate changes, alongside changing image size and batch size: {'ep':0,  'sz':128, 'bs':64}, {'ep':(0,6),  'lr':(lr,lr*2)}, {'ep':6, 'bs':128,}, {'ep':6, 'lr':lr*2}, {'ep':16, 'sz':224,'bs':64}, {'ep':16, 'lr':lr}, {'ep':19, 'bs':192, 'keep_dl':True}, {'ep':19, 'lr':2*lr/(10/1.5)}, {'ep':31, 'lr':2*lr/(100/1.5)}, {'ep':37, 'sz':288, 'bs':128, 'min_scale':0.5}, {'ep':37, 'lr':2*lr/100}, {'ep':(38,50),'lr':2*lr/1000}]",
+        "schedule": [
```

Comment:
This is minor, but by any chance can you express the learning rate, batch size, and image size schedule in a format similar to [#54](https://github.com/stanford-futuredata/dawn-bench-entries/pull/54)